cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -233,7 +233,12 @@ class UsingTest extends DocumentingTest {
             #However, it will also break transactional isolation and thus it should only be used where needed.""".stripMargin('#'))
       }
       note {
-        p("The <<query-use, `USE` clause>> can not be used together with the `PERIODIC COMMIT` clause.")
+        p("The <<query-use, `USE` clause>> can not be used together with the `PERIODIC COMMIT` query hint.")
+      }
+      note {
+        p(
+          """Queries with the `PERIODIC COMMIT` query hint can not be routed by <<operations-manual#causal-clustering-routing, Server-side routing>>.
+            |Such queries must rely on standard client-side routing, done by the Neo4j Driver.""".stripMargin)
       }
```

Comment:
Shouldn't this note also be in the periodic commit section (generated from `LoadCSVTest`)

Reply:
Nice catch. I added a line there as well.